cpp/util/fake_etcd.cc

```diff
@@ -157,7 +190,7 @@ void FakeEtcdClient::Get(const Request& req, GetResponse* resp, Task* task) {
   VLOG(1) << "GET " << req.key;
   const string key(NormalizeKey(req.key));
 
-  CHECK_LE(req.wait_index, 0) << "not implemented";
+  CHECK_NE(key, "/") << "not implemented";
```

Comment:
This is a weird edge case, and since nothing uses it, this will do for now. :stuck_out_tongue: